### PR TITLE
Auditd metadata and CHANGELOG update

### PR DESCRIPTION
**Pull request by rmoles**
Bumped cookbook version in metadata. Added change comments to CHANGELOG.

Signed-off-by: Ross Moles <rmoles@chef.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,32 @@
...
+## 2.3.4 (2018-11-05)
+
+- Disable dokken testing in travis for now
+- Simplify ignore files
+- Reenable dokken testing on Ubuntu
+- Add Travis badge to readme
+- Fix travis config
+- kitchen yaml updated to include cis config. CIS erb updated to fix control 4.1.17
+- added OS version check for control 4.1.9
+- tested updates
```
Comment (james-stocks) [inline CHANGELOG.md]: Not all these commits are worth documenting. I suspect these commits should have been rebased down to a lesser number of commits at the time they were PR'ed into the repo; I'm not sure if we can (or should try to) fix the commits retrospectively.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,32 @@
...
+- updated kitchen.yml. Support now added to edit the auditd.conf to bring it in line with CIS spec.
+- updated metadata
+- Fixed indentation issues. Added file permission checks for auditd.conf
+- Changed copyright message to chef. Removed commented lines in cis.auditd.conf.erb. Changed the platform version check in cis.rules.erb. Added an extra test in run_spec.rb.
+- made changes based on comments to make CIS conf changes optional
+- reverted metadata file. Changes to layout
+- added helper for auditd_conffile
+- re-added copyright in conf_file.rb based on comment
+
+## 2.3.4 (2018-11-5)
```
[the remark this answers is not in the document]
Comment (rmoles) [reply CHANGELOG.md]: yeah not sure what happened there.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
 This file is used to list changes made in each version of auditd.
 
+## 2.3.4 (2018-11-8)
+
+- Added CIS-rules kitchen suite
```
Comment (rachelrice) [inline CHANGELOG.md]: I would stick with one tense throughout - e.g add & update rather than added & updated - I think the former is preferred.

**Comment by rmoles** [comment]
Travis CI is not running. Merging as this PR has been approved.

**Pull request merged**